src/lib/components/glue/form/Multiselect.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+
+	type Option = {
+		value: string;
+		label: string;
+		canRemove: boolean;
+	};
+
+	export let options: Option[];
+	export let heading: string = '';
+	export let placeholder: string = '';
+	export let selectedOptions: Option[] = [];
+	export let showError: boolean = false;
+
+	let showDropdown = false;
+	let search = '';
+	let inputElement: HTMLInputElement;
+	let dropdownElement: HTMLDivElement;
+
+	function addOption(option: Option) {
+		if (!selectedOptions.some((o) => o.value === option.value)) {
+			selectedOptions = [...selectedOptions, option];
+		}
+		showDropdown = false;
+		search = '';
+	}
```

Comment:
nit: not a huge deal, but we're using arrow functions in all of our other files, so it would be good to make it consistent by using arrow functions here too 
`const addOption = (option: Option) => {}`